slurm.sh

```diff
@@ -272,6 +285,14 @@ slurm_monitor_job() {
         # complete, which we want before we kill the tail
         slurmLogStatus=$(slurm_completed_job_status_from_sacct "$jobStdout" "$jobId" "yes")
 
+        # Generate summary statistics
+        summaryStats=$( slurm_resource_usage_summary $slurmJobId )
+        if [[ -z "$summaryStats" ]]; then
+            warn "Warning: No SLURM job efficiency report generated." "$quiet"
```

Comment:
@irisdianauy  I think this will never be executed, because after removing `2> /dev/null ` in one of your previous commits, the variable `summaryStats` will never be empty. Is this correct?

Reply:
Yes if all nodes do throw the perl exception. This is what I'm not sure about.

Reply:
Yes if all nodes do throw a perl exception. Will remove redirection for now, so we can see other potential errors, as you pointed out in an earlier comment. Thanks again!